x-pack/plugins/security_solution/cypress/e2e/detection_rules/rule_snoozing.cy.ts

```diff
@@ -0,0 +1,296 @@
+    goToActionsStepTab();
+
+    expectRuleSnoozed('1 hours');
+  });
+
+  describe('Rules management table', () => {
+    it('snoozes a rule without actions for 3 hours', () => {
+      createRule(getNewRule({ name: 'Test rule without actions' }));
+
+      visitWithoutDateRange(SECURITY_DETECTIONS_RULES_URL);
+
+      snoozeRuleInTable({
+        tableSelector: RULES_MANAGEMENT_TABLE,
+        ruleName: 'Test rule without actions',
+        duration: '3 hours',
+      });
+
+      expectSnoozeSuccessToast();
+      expectRuleSnoozedInTable({
+        tableSelector: RULES_MANAGEMENT_TABLE,
+        ruleName: 'Test rule without actions',
+        duration: '3 hours',
+      });
+    });
+
+    it('snoozes a rule with actions for 2 days', () => {
+      createRuleWithActions({ name: 'Test rule with actions' }, createRule);
+
+      visitWithoutDateRange(SECURITY_DETECTIONS_RULES_URL);
+
+      snoozeRuleInTable({
+        tableSelector: RULES_MANAGEMENT_TABLE,
+        ruleName: 'Test rule with actions',
+        duration: '2 days',
+      });
+
+      expectSnoozeSuccessToast();
+      expectRuleSnoozedInTable({
+        tableSelector: RULES_MANAGEMENT_TABLE,
+        ruleName: 'Test rule with actions',
+        duration: '2 days',
+      });
+    });
+
+    it('unsnoozes a rule with actions', () => {
+      createSnoozedRule(getNewRule({ name: 'Snoozed rule' }));
+
+      visitWithoutDateRange(SECURITY_DETECTIONS_RULES_URL);
+
+      unsnoozeRuleInTable({
+        tableSelector: RULES_MANAGEMENT_TABLE,
+        ruleName: 'Snoozed rule',
+      });
+
+      expectUnsnoozeSuccessToast();
+      expectRuleUnsnoozedInTable({
+        tableSelector: RULES_MANAGEMENT_TABLE,
+        ruleName: 'Snoozed rule',
+      });
+    });
+
+    it('ensures snooze settings persist after page reload', () => {
+      createRule(getNewRule({ name: 'Test persistence' }));
+
+      visitWithoutDateRange(SECURITY_DETECTIONS_RULES_URL);
+
+      snoozeRuleInTable({
+        tableSelector: RULES_MANAGEMENT_TABLE,
+        ruleName: 'Test persistence',
+        duration: '3 days',
+      });
+
+      cy.reload();
+
+      expectRuleSnoozedInTable({
+        tableSelector: RULES_MANAGEMENT_TABLE,
+        ruleName: 'Test persistence',
+        duration: '3 days',
+      });
+    });
+
+    it('ensures a duplicated rule is not snoozed', () => {
+      createRule(getNewRule({ name: 'Test rule' }));
+
+      visitWithoutDateRange(SECURITY_DETECTIONS_RULES_URL);
+
+      duplicateFirstRule();
+
+      // Make sure rules table is shown as it navigates to rule editing page after successful duplication
+      visitWithoutDateRange(SECURITY_DETECTIONS_RULES_URL);
+
+      expectRuleUnsnoozedInTable({
+        tableSelector: RULES_MANAGEMENT_TABLE,
+        ruleName: 'Test rule',
+      });
+    });
+  });
+
+  describe('Rule editing page / actions tab', () => {
+    beforeEach(() => {
+      deleteConnectors();
+    });
+
+    it('ensures actions are visible', () => {
```

Comment:
Is this as a seperate test needed? It is not concretely related to snoozing.

If you want to have the test fail if the actions are not visible, you could move the `cy.get(actionFormSelector(0)).should('be.visible');` assertion to the next test ("adds an action to a snoozed rule"), which already contains the whole setup before the assertions.

Reply:
Good point. I've removed this test as the next one testing that it's possible to create new actions actually covers this case too.